### DISCUSSION TITLE
unRAID Docker Compose

### DISCUSSION THREAD

**Discussion by adamzvolanek** [nextcloud/all-in-one#4602] [Questions] [answered]
I've been attempting to convert from Nextcloud to Nextcloud AIO using docker compose on Unraid. I setup everything per the sprinkled aforementioned unRAID instructions and created my [docker-compose](https://github.com/adamzvolanek/DevRack/blob/main/docker-compose/cloud-aio/cloud-aio.yaml) file.

However upon stopping the stack, I'm noticing plenty of Nextcloud all in one containers lingering around in a stopped mode. After a reboot of my test server and attempting to start the compose stack, I am unable to access my Nextcloud as previously running on my `subdomain.domain.tld` but instead it is asking for the initial setup password again.

Any suggestions or reasons why this is occurring?

**Comment by szaimen** [comment, marked as answer]
Hi, the reason is that you always have to stop the containers via the aio interface on port 8080. Apart from that see https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-stopstartupdate-containers-or-trigger-the-daily-backup-from-a-script-externally
